src/server.rs

```diff
@@ -254,19 +254,25 @@ struct Cli {
 fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
     let data_dir = util::get_default_data_dir();
 
-    // Initialize the logger
-    let log_writer = util::init_logger(&data_dir)?;
-
-    // Initialize the logger and log the log file path
-    log::info!("Logging initialized. Log file located at: {:?}", data_dir.join("debug.log"));
-
+    // Read the configuration file
     let contents = fs::read_to_string("../../example-config.toml")
```

Comment:
Here I think you need to concatenate with the data_dir path yielded by `get_default_data_dir()` otherwise you’re dependent from which current working directory path `civkitd` is launched.